R/iSEEinit.R

```diff
@@ -71,6 +70,18 @@ iSEEinit <- function(sce,
          "Please select one of these: ",
          paste(available_reddims, collapse = ", "))
   }
+  
+  if (is.data.frame(features)) {
+    features <- as.character(features[["id"]])
+  } else if (is.list(features)) {
+    features <- as.character(unlist(features))
```

Comment:
it is a bit hard to check on the "real content" of a list. 
When would we be encountering this use case?

Reply:
that is absolutely a fair point! tbh I don't know any concrete case that gives back a list, I just added it to have more input formats. But in reality I think that the df format would be the most useful, because people usually store these things in tables, and when they extract the genes column they would still be able to use the function, whether they get back a df or a vector. Eventually, I think I will remove the list option and leave it simple

Reply:
Yep, it sounds like a sensible decision. 
I thought you had some other options in mind where a list would be returned.

But if that is not the case, we might be on the safer side if we don't need to support that input being checked

Reply:
now the feature can be either a vector or df [4dd6bf1](https://github.com/NajlaAbassi/iSEEfier/pull/10/commits/4dd6bf144692f48a558ea9589a788dd16ede03e7)